composer.json

```diff
@@ -22,7 +22,8 @@
         "phpunit/phpunit": "^9.5",
         "symfony/form": "^6.2",
         "symfony/twig-bridge": "^6.2",
-        "twig/twig": "^3.5"
+        "twig/twig": "^3.5",
+        "phpstan/phpstan": "^1.9"
```

Comment:
I was thinking of installing "vimeo/psalm" instead as that's the static tool used by the moneyphp library

Reply:
Fair point!